src/Slot-Core/ComposedSlot.class.st

```diff
@@ -0,0 +1,60 @@
+"
+Slots can be composed. A composition always has one ""real"" slot that knows how to store values and one ore more SlotDecorators. The decorators are chained and delegate reading/writing to the front of the compostion until the finally use the one storage slot to store or read the value.
+"
+Class {
+	#name : #ComposedSlot,
```

Comment:
Why the comment talks about `SlotDecorators` and this one is called `ComposedSlot`? Looking at the implementation `SlotDecorator` looks better..

Reply:
removed for now, SlotDecorator is really better